packages/jest-circus/src/__tests__/tsconfig.json

```diff
@@ -3,5 +3,6 @@
   "compilerOptions": {
     "rootDir": "../"
   },
-  "include": ["../**/*"]
+  "include": ["../**/*"],
+  "references": [{"path": "../../"}]
```

Comment:
To typecheck test files, `tsc` runs with `-b` flag. Also it gets files in `__tests__` and `src` directories still in TypeScript, not the compiled ones from `build`. It has to build them (in memory). Logically `src` has to be build first, so pointing references to `packages/jest-circus/tsconfig.json` makes sense.